tests/providers/google/cloud/operators/test_cloud_run.py

```diff
@@ -387,3 +393,27 @@ def test_execute_with_invalid_limit(self, hook_mock):
         limit = -1
         with pytest.raises(expected_exception=AirflowException):
             CloudRunListJobsOperator(task_id=TASK_ID, project_id=PROJECT_ID, region=REGION, limit=limit)
+
+
+class TestCloudRunCreateServiceOperator:
+    def test_template_fields(self):
+        operator = CloudRunCreateServiceOperator(
+            task_id=TASK_ID, project_id=PROJECT_ID, region=REGION, service_name=SERVICE_NAME, service=SERVICE
+        )
+
+        _assert_common_template_fields(operator.template_fields)
+        assert "service_name" in operator.template_fields
+
+    @mock.patch(CLOUD_RUN_SERVICE_HOOK_PATH)
+    def test_create(self, hook_mock):
```

Comment:
We usually name this test as test_execute()

Reply:
done